cx_Freeze/hooks.py

```diff
@@ -370,13 +370,14 @@ def pycryptodome_filename(dir_comps, filename):
 
 def load__ctypes(finder: ModuleFinder, module: Module) -> None:
     """
-    In Windows, the _ctypes module in Python 3.8+ requires an additional dll
-    libffi-7.dll to be present in the build directory.
+    In Windows, the _ctypes module in Python 3.8+ requires an additional
+    libffi dll to be present in the build directory.
     """
     if WIN32 and sys.version_info >= (3, 8) and not MINGW:
-        dll_name = "libffi-7.dll"
-        dll_path = Path(sys.base_prefix, "DLLs", dll_name)
-        finder.IncludeFiles(dll_path, Path("lib", dll_name))
+        dll_pattern = "libffi*-.dll"
```

Comment:
Would not be `dll_pattern = "libffi-*.dll"`?

Reply:
You're right, well spotted! That'll teach me to try and make last second changes..